src/assets/Standards/CollectibleStandards/ERC1155/ERC1155Standard.test.ts

```diff
@@ -12,13 +12,38 @@ const ERC1155_ADDRESS = '0xfaaFDc07907ff5120a76b34b731b278c38d6043C';
 describe('ERC1155Standard', () => {
   let erc1155Standard: ERC1155Standard;
   let web3: any;
+  nock.disableNetConnect();
```

Comment:
Since you have `nock.restore` in an `afterAll`, should you put this in a `beforeAll`?

Reply:
Sure!

Reply:
accidentally amended instead of making a new commit, but its addressed here now: https://github.com/MetaMask/controllers/pull/667/commits/3e48605dcb64c65ac23b6d21d10f06cbf2cb6d1a